### DISCUSSION TITLE
Any plans to include Zhuyin?

### DISCUSSION THREAD

**Discussion by RJacobs-coder** [fcitx/fcitx5#465] [Ideas]
Hello All,

Just downloaded fcitx5 and loving the Chinese / Japanese inputs provided but I currently live in Taiwan and the last thing stopping me from deploying Arch Linux to my entire work is a working Zhuyin Input Method that uses the traditional Chinese (BOPOMOFO keyboard layout)

Is there a plan to include this? 
I did find one of github but could not get it working, looking for alternatives also.

All the best

**Comment by wengxt** [comment]
fcitx5-chewing / fcitx5-zhuyin are both available.
